docs/approach.rst

```diff
@@ -243,7 +249,9 @@ in a dimensionally reduced version of the dataset which is then used in the
 :ref:`TEDICA` step.
 
 .. image:: /_static/a12_pca_reduced_data.png
-
+.. _AIC: https://en.wikipedia.org/wiki/Akaike_information_criterion
+.. _KIC: https://en.wikipedia.org/wiki/Kullback%E2%80%93Leibler_divergence#Relationship_between_models_and_reality
```

Comment:
I think we probably want to link to the top of the article, rather than this subsection.

Reply:
Sure, whatever you think will be more meaningful for our users.

Reply:
Is there something in particular you think would be valuable from that subsection? If so we could either add it to the docs or add a second link to it to highlight it.

Reply:
I wanted to highlight the relationship with AIC. I could link to the main page and then write a short sentence that says `you can see how it relates to AIC here.` with the link to that particular section.

Reply:
That would be great!

Reply:
The latest commit deals with this. Thank you for the feedback!